components/PlayerDetails/index.js

```diff
@@ -3,29 +3,123 @@ import Image from "next/image";
 
 export default function PlayerDetails({ player }) {
   if (player?.response?.length > 0) {
-    return (
-      <StyledContainer>
-        <StyledList>
-          <StyledListItem>{player.response[0].player.name}</StyledListItem>
-          <StyledListItem>
-            {player.response[0].statistics[0].team.name}
-          </StyledListItem>
-          <StyledListItem>
-            Season: {player.parameters.season}/
-            {parseInt(player.parameters.season, 10) + 1}
-          </StyledListItem>
-          <StyledListItem>
-            Goals: {player.response[0].statistics[0].goals.total}
-          </StyledListItem>
-        </StyledList>
-        <StyledImage
-          src={player.response[0].player.photo}
-          height={80}
-          width={80}
-          alt={player.response[0].player.name}
-        />
-      </StyledContainer>
-    );
+    if (player.response[0].statistics[0].games.position === "Goalkeeper") {
+      return (
+        <StyledContainer>
+          <StyledList>
+            <StyledListItem>{player.response[0].player.name}</StyledListItem>
+            <StyledListItem>
+              {player.response[0].statistics[0].team.name}
+            </StyledListItem>
+            <StyledListItem>
+              Nationality: {player.response[0].player.nationality}
+            </StyledListItem>
+            <StyledListItem>
+              Position: {player.response[0].statistics[0].games.position}
+            </StyledListItem>
+            <StyledListItem>
+              Season: {player.parameters.season}/
+              {parseInt(player.parameters.season, 10) + 1}
+            </StyledListItem>
+            <StyledListItem>
+              Appearances:{" "}
+              {player.response[0]?.statistics[0]?.games?.appearences ?? "N/A"}
+            </StyledListItem>
+            <StyledListItem>
+              Goals conceded:{" "}
+              {player.response[0]?.statistics[0]?.goals?.conceded ?? "N/A"}
+            </StyledListItem>
+            <StyledListItem>
+              Goal saves:{" "}
+              {player.response[0]?.statistics[0]?.goal?.saves ?? "N/A"}
+            </StyledListItem>
```

Comment:
I think you're repeating quite a bit of code (lines 7 - 27). You could possibly move the if statements to the statistics that are actually different and leave details such as "name", "team", "nationality" etc. for all players. 

```
{player.response[0].statistics[0].games.position === "Goalkeeper" ? <StyledListItem>
              Goals conceded:{" "}
              {player.response[0]?.statistics[0]?.goals?.conceded ?? "N/A"}
            </StyledListItem> : ...}
```

That would save a few lines of code. :)

Reply:
Good point, I've refactored my code accordingly.